.github/workflows/release.yml

```diff
@@ -1,6 +1,12 @@
 on: 
   release:
     types: [created]
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tag to build and upload artifact bundle for'
+        required: true
+        default: ${{ github.ref_name }}
```

Comment:
If this value is revision, can we update Zip files to the actual release?

Reply:
I think we can't upload in that case (for commit hashes).